src/index.html

```diff
@@ -10,23 +10,26 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="search-bar">
+      <input
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+        class="search-bar__big input"
+      />
```

Comment:
HTML Formatting issue: The attributes for the input element are not correctly formatted. Each attribute should start on a new line with 2-space indentation related to the tag. The tag’s closing bracket should be on the same level as the opening one. Please refer to the checklist for the correct format.